message_ix/tests/test_integration.py

```diff
@@ -70,9 +77,13 @@ def test_run_remove_solution(test_mp, request):
     )
 
 
-def test_shift_first_model_year(test_mp, request):
+# TODO IXMP4Backend doesn't support clone(shift_first_model_year=...) yet
```

Comment:
I don't think this should be the responsibility of ixmp, as that is a stack level violation. See #873 and #254.